pkg/dogstatsd/server.go

```diff
@@ -341,6 +347,39 @@ func NewServer(demultiplexer aggregator.Demultiplexer, serverless bool) (*Server
 		}
 	}
 
+	// Creating a new logger
+	cfg := seelogCfg.NewSeelogConfig(
+		"dogstatsd",
+		"info",
+		"common",
+		"",
+		"%Date %Time | [DOGSTATSD_STATS] | [%LEV] %Msg ",
```

Comment:
Since the log file has only one type of entries, do we really need `| [DOGSTATSD_STATS] | [%LEV]` part?

Reply:
This might be for more future proofing but I guess for now its not needed, i'll remove it instead then